packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -384,13 +385,34 @@ export abstract class SharedObjectCore<
 	 * @param local - True if the shared object is local
 	 * @param localOpMetadata - For local client messages, this is the metadata that was submitted with the message.
 	 * For messages from a remote client, this will be undefined.
+	 *
+	 * @deprecated - Replaced by processMessagesCore.
```

Comment:
Note: `@deprecated` comments don't want the `-` between the tag and the message. That is unique to `@param` and `@typeParam` comments (because their syntax supports type annotations).
```suggestion
	 * @deprecated Replaced by {@link SharedObjectCore.processMessagesCore}.
```